tests/test_writer.py

```diff
@@ -205,6 +226,7 @@ def test_delimited_dump():
         }
     ]
 
+    # remove URLs from list_tags and give it a custom delimiter
     text_output = rispy.dumps(entries, list_tags=["AU"], delimiter_mapping={"UR": ","})
```

Comment:
variable name question - what about `list_tags` and `delimiter_tags_mapping`? That makes it clearer (to me at least) that you should put your variables in one place or the other?

Reply:
I can do that, I think the original reason I put `delimiter_mapping` was because the other tag mapping is just `mapping`